ember-flight-icons/index.js

```diff
@@ -2,4 +2,32 @@
 
 module.exports = {
   name: require('./package').name,
+
+  contentFor(type) {
+    if (type === 'body-footer') {
+      return `
+        <script>
+          (function() {
+            var req = new XMLHttpRequest();
+
+            req.addEventListener('load', function(event) {
+              var src = event.target.responseText;
+              var div = document.createElement('div');
+
+              div.innerHTML = src;
+
+              var svg = div.querySelector('svg');
+
+              document.body.appendChild(svg);
+            });
+
+            // BLOCKER: How to get at the sprite _here_
+            req.open('GET', '/@hashicorp/ember-flight-icons/icons/sprite.svg');
+
+            req.send();
+          })();
+        </script>
```

Comment:
👋  not quite sure what the overall thinking is here, but if you are going forwards with some sort of inline script, Consul (and Vault/CloudUI also probably and I would imagine other products in the future) doesn't allow inline scripts for security reasons. So if this is the direction you are going then this will need to be an external script if possible.

Reply:
Thanks for the heads up on this. We're going to try something slightly different to attempt to avoid this.